core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -484,6 +502,50 @@ void core_init_mmu_prtn(struct mmu_partition *prtn, struct tee_mmap_region *mm)
 	}
 }
 
+/*
+ * TAs page table is an entry inside level 1 page table.
+ *
+ * If base table level is 1 - user_va_idx is already the index,
+ *                            so nothing to do.
+ * If base table level is 0 - user_va_idx is an level 0 index,
+ *                            so we need to create a level 1 table and TA entry
+ *                            will be the first entry there.
+ */
+static void core_init_mmu_prtn_ta(struct mmu_partition *prtn __maybe_unused)
+{
+#if (CORE_MMU_BASE_TABLE_LEVEL == 0)
```

Comment:
We normally try to avoid `#if` and `#ifdef` inside function, but this is OK since it's the entire body of the function.